Makefile

```diff
@@ -18,8 +19,12 @@ cache = $(if $(cached-$1),,$(eval cached-$1 := 1)$(eval cache-$1 := $($1)))$(cac
 $(BINARY): vendor $(SRC)
 	go build ${LDFLAGS}
 
+$(AUTOPATH):
+	mkdir -p /etc/bash_completion.d/
+	sudo cp vendor/github.com/urfave/cli/autocomplete/bash_autocomplete $(AUTOPATH)
```

Comment:
This target has a dependency on vendor.  If you try to run the "build" it without vendor first it can fail.
```
make /etc/bash_completion.d/rita
mkdir -p /etc/bash_completion.d/
sudo cp vendor/github.com/urfave/cli/autocomplete/bash_autocomplete /etc/bash_completion.d/rita
cp: cannot stat 'vendor/github.com/urfave/cli/autocomplete/bash_autocomplete': No such file or directory
Makefile:23: recipe for target '/etc/bash_completion.d/rita' failed
make: *** [/etc/bash_completion.d/rita] Error 1
```

I'm not sure if there's a benefit having this be its own build target.  How would you feel about just moving these `mkdir` and `cp` instructions under the existing `install` target?

Reply:
I wanted it to be its own target because moving the autocomplete file requires root access. I can just add `vendor` as a prerequisite of `$(AUTOPATH)` and that should fix this.

Reply:
Makes sense, thanks!